src/routes.json

```diff
@@ -205,11 +205,11 @@
         "title": "Scratch 3.0 Preview"
     },
     {
-        "name": "3faq",
-        "pattern": "^/3faq/?$",
-        "routeAlias": "/3faq/?$",
-        "view": "preview-faq/preview-faq",
-        "title": "Scratch 3.0 FAQ"
+        "name": "parents",
+        "pattern": "^/parents/?$",
+        "routeAlias": "/parents/",
+        "view": "parents/parents",
+        "title": "For Parents"
```

Comment:
What's going on here?

Reply:
It fixed a an earlier conflict with a force push.  You and I just solved this by making the 'master' push into develop